tests/cases/with/out

```diff
@@ -2,33 +2,50 @@
   (with b; c)
   (
     with b;
-    /*
-     b
-     */
-    c
+      /*
+       b
+       */
+      c
   )
   (
     with
     /*
      a
      */
-    b;
-    c
+    b; c
   )
   (
     with
     /*
      a
      */
     b;
-    /*
-     b
-     */
-    c
+      /*
+       b
+       */
+      c
   )
   (with b; cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc)
   (
-    with b;
-    cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+    with b; cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
   )
+  { a = with b; 1; }
+  { a = with b; 1 + 1; }
+  { a = with b; { c = 1; }; }
+  {
+    a = with b; {
+      c = 1;
+      d = 2;
+      e = 3;
+    };
+  }
+  {
+    a = with b;
+      # comment
+      1;
+  }
+  {
+    a = with b; 1;
+    # comment
+  }
```

Comment:
As a whole this is a very good change:

```diff
--- a/pkgs/applications/accessibility/contrast/default.nix
+++ b/pkgs/applications/accessibility/contrast/default.nix
@@ -65,12 +65,10 @@ stdenv.mkDerivation rec {
     patchShebangs build-aux/meson_post_install.py
   '';

-  meta =
-    with lib;
-    {
-      description = "Checks whether the contrast between two colors meet the WCAG requirements";
-      homepage = "https://gitlab.gnome.org/World/design/contrast";
-      license = licenses.gpl3;
-      maintainers = with maintainers; [ jtojnar ];
-    };
+  meta = with lib; {
+    description = "Checks whether the contrast between two colors meet the WCAG requirements";
+    homepage = "https://gitlab.gnome.org/World/design/contrast";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ jtojnar ];
+  };
```

I would be perfect if we:
- allowed only 1 with to be inlined
- removed the bin-op from the PR

Those seem to be simple changes to made

Thanks for the contribution!